### DISCUSSION TITLE
Best way to get Neorg default keybindings showing up?

### DISCUSSION THREAD

**Discussion by xfzv** [mrjones2014/legendary.nvim#407] [Q&A] [answered]
[Neorg](https://github.com/nvim-neorg/neorg/) comes with default [keybindings](https://github.com/nvim-neorg/neorg/blob/7d507d4fdd1c4a3cb8a8299a469d22190ebaf7f0/lua/neorg/modules/core/keybinds/keybinds.lua), which are using the `localleader` key.

I have the latter mapped to `space` (like `leader`) but these keybindings are nowhere to be seen in `Legendary`.

Is there a way to make `Legendary` pick them or would I have to manually add them in my config?

Thanks.

**Comment by mrjones2014** [comment]
I'm not familiar with neorg's API but if there's a way to get the key mappings via an API you could write an [extension](https://github.com/mrjones2014/legendary.nvim/blob/master/doc/EXTENSIONS.md) for legendary. I can take a look tomorrow morning.

**Reply by mrjones2014** [reply, marked as answer]
I can't see a way to get current keybinds from API.

**Reply by xfzv** [reply]
Thanks for the heads up! That's something that could be improved on `Neorg` side I guess.
